client/signup/config/flows.js

```diff
@@ -157,13 +159,16 @@ function removeUserStepFromFlow( flow ) {
 	} );
 }
 
-function getCurrentFlowNameFromTest() {
-	// No tests currently running
+function getCurrentFlowName( currentUrl ) {
+	// Headstart test - Only consider users from the general /start path.
+	if ( '/start' === currentUrl && 'headstart' === abtest( 'headstart' ) ) {
```

Comment:
oh, good point! Hm. I'll have to look, but I wonder if this function gets called for flows other than the default. Like, if you go to `/start/jetpack`, does this function get called? If not, we can probably just avoid testing the current URL. But if it *does* get called, we'll want some check in here to prevent users in other flows from ending up in the abtest.